modules/tinymce/src/core/test/ts/browser/EditorRemoveTest.ts

```diff
@@ -44,7 +44,7 @@ UnitTest.asynctest('browser.tinymce.core.EditorRemoveTest', (success, failure) =
       cAssertTextareaDisplayStyle('none'),
       cRemoveEditor,
       cAssertTextareaDisplayStyle(''),
-      Chain.op((editor) => EditorManager.init({ selector: '#tinymce' })),
+      Chain.op((editor) => {EditorManager.init({ selector: '#tinymce' });}),
```

Comment:
This should probably have spacing or be on a new line?

Reply:
The lint will catch it when it's re-enabled (I'm getting there!). But I suppose I can do it manually...